draft-ietf-tcpm-converters.mkd

```diff
@@ -283,6 +291,11 @@ with the names and/or the addresses of one or more Transport Converters.
 
 The architecture does not mandate anything on the server side.
 
+<!---
+I would explain a bit more why this does not impact TLS. Maybe just saying
+that TLS sits in the bytestream, after the Convert Protocol (I don't know
+what "address sharing mechanisms" refers to).
```

Comment:
Address sharing is  used to refer to mechanisms to share IP address/prefixes. This is not only about NATs, but also other mechanisms such us NPTv6p, etc. 

This term is well known in the IETF.